pkg/tracing/lightstep/lightstep.go

```diff
@@ -0,0 +1,59 @@
+package lightstep
+
+import (
+	"context"
+	"errors"
+	"io"
+
+	"github.com/lightstep/lightstep-tracer-go"
+	"github.com/opentracing/opentracing-go"
+	"gopkg.in/yaml.v2"
+)
+
+// Config - YAML configuration.
+type Config struct {
+	// AccessToken is the unique API key for your LightStep project.  It is
+	// available on your account page at https://app.lightstep.com/account
+	AccessToken string `yaml:"access_token"`
+
+	// Collector is the host, port, and plaintext option to use
+	// for the collector.
+	Collector lightstep.Endpoint `yaml:"collector"`
+}
+
+// Tracer wraps the Lightstep tracer and the context.
+type Tracer struct {
+	lightstep.Tracer
+	ctx context.Context
+}
+
+// Close synchronously flushes the Lightstep tracer, then terminates it.
+func (t *Tracer) Close() error {
+	lightstepTracer := t.Tracer
```

Comment:
```suggestion
```

Reply:
Why it was resolved? I think @povilasv had a good point, we could just do `t.Tracer.Close(t.ctx)` (:

Reply:
Ermmmm you know what, I resolved it before updating the code and ended up missing it. Applying now!